wandb/sdk/wandb_run.py

```diff
@@ -731,7 +732,7 @@ def _handle_launch_artifact_overrides(self) -> None:
             and self._settings.launch_config_path
             and os.path.exists(self._settings.launch_config_path)
         ):
-            self._save(self._settings.launch_config_path)
+            self.save(self._settings.launch_config_path)
```

Comment:
omg, it is duplicated in init and here 🤦

Reply:
yep! can't assume that private methods aren't accessed... scary